dev/com.ibm.ws.mongo_fat/fat/src/com/ibm/ws/mongo/fat/MongoDefaultSSLTest.java

```diff
@@ -38,10 +38,11 @@ public static void beforeClass() throws Exception {
 
     @AfterClass
     public static void afterClass() throws Exception {
-        server.stopServer("SRVE0777E.*com.mongodb.CommandResult",
```

Comment:
They (and all of the others) were removed intentionally because they're not actually thrown in the server log.  I can load up an ancient RTC workspace and see if I can figure out why all of this stuff was added...